tools/data-handler/src/containers/template.ts

```diff
@@ -64,10 +64,10 @@ export class Template extends CardContainer {
     },
   ];
 
-  constructor(path: string, template: Resource, project: Project) {
+  constructor(project: Project, template: Resource,) {
     // Templates might come from modules. Remove module name from template name.
     const templateName = basename(template.name);
-    super(path, templateName);
+    super(template.path!, templateName);
```

Comment:
Why is path optional in the first place?

Reply:
I think - not sure - the reason is that in some cases when `Resource` is handled, the path is not known and not really necessary. And putting it into the instance of resource would be extra work with no benefits.

Reply:
I'll make a ticket to make it mandatory (don't want to muddle this PR). 
In some cases when templates are created, it is not necessary to have `path` (as `Template` can figure it out itself. But we can always pass empty string and all is well.

Reply:
https://cyberismo.atlassian.net/browse/INTDEV-535

Reply:
Could it be even more clear, if we had separate types for that:
ResourceInfo(what would a better name?) = Omit<Resource, 'path'>

Reply:
`Resource` only has `name` and `path`, so in effect it would be `ResourceName`. 
`ResourceInfo` would be fine name if we expect `Resource` to be expanded in the future.